### DISCUSSION TITLE
GitHub Pages does not show site

### DISCUSSION THREAD

**Discussion by Sven72** [community/community#22585] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I setted up a GitHub Pages Site (hopefully correct), pushed my code with the desktop App within a folder named like this </p>
<pre><code>[username].github.io
</code></pre>
<p>and nothing happened, when I click on “environment” -&gt; “view deployment”</p>
<p>It would be great if someone could help.</p>
<p>Link to repo</p>
<p><a href="https://github.com/Sven72/Sven72.github.io" rel="nofollow noopener">https://github.com/Sven72/Sven72.github.io</a></p>
<p>And link to page</p>
<p><a href="https://sven72.github.io/" rel="nofollow noopener">https://sven72.github.io/</a></p>
<p>Cheers,</p>
<p>S</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/sven72">@sven72</a>,</p>
<p>Thanks for being here! I see your page is serving on <a href="https://sven72.github.io/index.html" rel="nofollow noopener">https://sven72.github.io/index .html</a>, notice the extra space in index .html typo? fixing that should take care of the rest. I hope this helps!</p>
<p>Best,</p>
<p>Andrea</p>
